scripts/train_model.py

```
@@ -46,50 +24,104 @@
-        print(f"DEBUG: Left hand normalization failed", file=sys.stderr)
-        return None
-    if right is None:
-        # Allow single-hand data - just use zeros for the missing hand
-        right = np.zeros_like(pts[:21])
+        pts = np.array(lm).reshape(-1, 3)
 
-    return np.concatenate([left, right]).flatten()
+    num_pts = len(pts)
+    
+    def _norm_geometric(points: np.ndarray, ref_idx: int, scale_indices: Tuple[int, int]) -> np.ndarray:
+        if len(points) <= ref_idx: return points
```

Ruff (0.14.10)

Multiple statements on one line (colon)

(E701)